src/main/java/org/biouno/unochoice/AbstractScriptableParameter.java

```diff
@@ -134,6 +133,7 @@ protected AbstractScriptableParameter(String name, String description, String ra
         String projectName = null;
         String projectFullName = null;
         if (currentRequest != null) {
+            // TODO: Check if we really need AbstractItem instead of Job class
```

Comment:
@kinow Same for this commentary: why do we use `AbstractItem` here but then filter with `Project` class in `getHelperParameters()`?

Reply:
From memory, the name or full name may have the project name. Or one of them may contain a folder name. Or a special object name (e.g. another plug-in that provides an `AbstractItem`.).

In case where the projectName is the name of an `AbstractItem` that is not a `Project`, we ignore that as we don't want to put another object instance in the `jenkinsProject` bound variable.